documentation/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -673,6 +671,38 @@ public void test_jpql_api_list_example() {
 		});
 	}
 
+	@Test
+	public void test_jpql_api_stream_example() {
+		doInJPA( this::entityManagerFactory, entityManager -> {
+			//tag::jpql-api-stream-example[]
+			try(Stream<Person> personStream = entityManager.createQuery(
+				"select p " +
+				"from Person p " +
+				"where p.name like :name", Person.class )
+			.setParameter( "name", "J%" )
+			.getResultStream()) {
+				List<Person> persons = personStream
+					.skip( 5 )
+					.limit( 5 )
+					.collect( Collectors.toList() );
+			}
+			//end::jpql-api-stream-example[]
+
+			// tag::jpql-api-stream-terminal-operation[]
+			List<Person> persons = entityManager.createQuery(
+					"select p " +
+					"from Person p " +
+					"where p.name like :name", Person.class )
+			.setParameter( "name", "J%" )
+			.getResultStream()
+			.skip( 5 )
+			.limit( 5 )
+			.collect( Collectors.toList() );
```

Comment:
It would be great to make consistent usage of code style. I even created a ticket to fix the various format issue (sometimes indentation produces really bad result because of mixed usage of TAB and space, e.g.).
Reformatting our testing code with the https://github.com/hibernate/hibernate-ide-codestyles will solve the issue easily.
So I am okay with this (there is some minor format issue above like Line 678. There should be a space after `try` and before `{`).
Many thanks for updating the User Guide, however.